driver/controller.go

```diff
@@ -191,6 +197,19 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if snapshot != nil && volumeReq.SizeGigaBytes > int64(snapshot.SizeGigaBytes) && volumeReq.SizeGigaBytes > 1 {
+		_, _, err := d.storageActions.Resize(ctx, vol.ID, int(volumeReq.SizeGigaBytes), volumeReq.Region)
+		log.Info("resizing volume because its requested size is larger than the size of the backing snapshot")
```

Comment:
Can we move this above the `Resize` call given it's "announcing" the possibly long-running operation?

Reply:
Moved it up one line.